### PR TITLE
Add memory tag and use it

### PR DESCRIPTION
**Pull request by djs55**

### CODE REVIEW

File: _data/xenapi.json [hunk 1]
```diff
@@ -11847,8 +11855,7 @@
 "lifecycle": [ 
   { "transition": "published",
     "release": "cowley",
-    "description": "A tunnel for network traffic" } ],
-"tag": "networking" }, { "name": "PCI",
```
Comment (robhoes) [inline _data/xenapi.json]: This patch currently reverts a few recent changes I made. Using xapi-project/xen-api#2185 to generate xenapi.json will fix this.

**Comment by robhoes** [comment]
Is memory-accounting.md the right place to add the list of XenAPI calls etc, or would it fit better in a page about memory under the XenAPI heading in the navbar?

**Comment by djs55** [comment]
My plan is to move this to xen-api, and put the DMC design mentioned on the
list in the design section

**Comment by robhoes** [comment]
I see, good idea.

**Comment by djs55** [comment]
Obsoleted by #107

**Pull request closed without merging**